src/directory-sync/interfaces/list-directories-options.interface.ts

```diff
@@ -2,5 +2,6 @@ import { PaginationOptions } from '../../common/interfaces/pagination-options.in
 
 export interface ListDirectoriesOptions extends PaginationOptions {
   domain?: string;
+  organization?: string;
```

Comment:
This should be `organization_id` to match what the API is expecting:

```suggestion
  organization_id?: string;
```